utils/generate_mbox.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+# You can use this script to create a test mbox with specific mail size
+# distribution.
+#
+# Running the script creates a 'testmbox' output file in the current directory.
+
+from email.mime.text import MIMEText
+from email.utils import formatdate
+import mailbox
+import random
+import string
+import os
+
+# Size distribution can be configured here. By default, a mbox with 5 mails of
+# 10kB, 80kB, 150kB and 250kB each in a randomized order are created.
+# ImapTest iterates through the mbox sequentially, so all randomness must be
+# in the mbox file.
+size_distribution = {
+  10:    5,
+  80:    5,
+  150:   5,
+  250:   5,
+  15000: 0,
+}
+
+mailfrom = 'sender@example.com'
+mailto = 'recipient@example.com'
+mbox_out = 'testmbox'
```

Comment:
could this be argv[1] and default to 'testmbox'?

Reply:
Why?  Renaming the output file is trivial.

Reply:
fair enough.